mail/views_test.py

```diff
@@ -114,28 +115,33 @@ class FinancialAidMailViewsTests(APITestCase):
     @classmethod
     def setUpTestData(cls):
         super().setUpTestData()
-        cls.financial_aid_mail_url = reverse('financial_aid_mail_api')
         # create a user with a role for one program
         with mute_signals(post_save):
             staff_profile = ProfileFactory.create()
-            cls.staff = staff_profile.user
+            cls.staff_user = staff_profile.user
         cls.program = ProgramFactory.create(live=True)
         Role.objects.create(
-            user=cls.staff,
+            user=cls.staff_user,
             program=cls.program,
             role=Staff.ROLE_ID
         )
+        cls.financial_aid = FinancialAidFactory.create(
+            tier_program=TierProgramFactory.create(program=cls.program)
+        )
+        cls.financial_aid_mail_url = reverse(
+            'financial_aid_mail_api',
+            kwargs={'financial_aid_id': cls.financial_aid.id}
+        )
         cls.request_data = {
             'email_subject': 'email subject',
             'email_body': 'email body',
-            'email_recipient': 'a@example.com'
         }
 
     def test_send_financial_aid_view(self, mock_mailgun_client):
```

Comment:
Can you add a test for the case where the user doesn't have permission to send the email for a given `FinancialAid`?